src/services/StructuralMetadataUtils.js

```diff
@@ -563,7 +563,7 @@ export default class StructuralMetadataUtils {
    * Insert a new timespan as child of an existing heading
    * @param {Object} obj - object of form values submitted
    * @param {Array} allItems - The entire structured metadata collection
-   * @returns {Array} - The updated structured metadata collection, with new object inserted
+   * @returns ({Object}, {Array}) - (New span, The updated structured metadata collection, with new object inserted)
    */
   insertNewTimespan(obj, allItems) {
     let clonedItems = [...allItems];
```

Comment:
So this is the old way that I mistakenly was using for cloning our total data structure.  The proper way to do it, is using the lodash helper method. I'm going to push up a commit with the fix in it.   This could cause oddball errors in the future.

Reply:
Actually, would you mind updating all similar occurrences `let clonedItems = [...allItems];` with `let clonedItems = cloneDeep(allItems)`?   You'll need to import the lodash method like you did in the test.  Thanks!

Reply:
Yeah sure! :)